ClassesHomework/Animal.cs

```diff
@@ -7,8 +7,54 @@
 namespace ClassesHomework
 {
     // write your Animal class here. 
-    //class Animal
-    //{
-        
-    //}
+    public class Animal
+    {
+        public Animal()
+        {
+
+        }
+        public Animal(string breed)
+        {
+            Breed = breed;
+
+        }
+        public Animal(int age)
+        {
+            Age = age;
+
+        }
+        public Animal(string breed, int age)
+        {
+            Breed = breed;
+            Age = age;
+
+        }
+        public bool Sit()
+        {
+            IsSitting = true;
+            return IsSitting;
+        }
+        public bool Stand()
+        {
+            IsSitting = false;
+            return IsSitting;
+        }
+
+        public int CelebrateBirthday()
+        {
+            Age = ++Age;
```

Comment:
This is redundant. The increment operator (`++Age` or `Age++`) modifies `Age`, so there's no need to also do an assignment.